styles/pypi/catppuccin.user.css

```diff
@@ -243,7 +248,9 @@
     }
 
     .dropdown__content {
-      border: 1px solid @accent-color !important;
```

Comment:
The "use border-color" instead message is telling you to only set the border-color, since Catppuccin userstyles only change the appearance and not the layout. So basically we don't want the other `border-style` and `border-width` that you added.